pyseext/field_helper.py

```diff
@@ -202,13 +204,17 @@ def get_field_value(self, form_cq: str, name: str) -> Any:
         Args:
             form_cq (str): The component query that identifies the form panel in which to look for the field
             name (str): The name of the field
+            is_date (bool): Indicates whether the field is a date time or not. Defaults to False.
 
         Returns:
             Any: The value of the field, or None if not found.
         """
         script = self._GET_FIELD_VALUE_TEMPLATE.format(form_cq=form_cq, name=name)
         self.ensure_javascript_loaded()
-        return self._driver.execute_script(script)
+
+        value = self._driver.execute_script(script)
```

Comment:
No need for this local now.

Sorry, picky me!!

Reply:
Although saying that, doesn't this need to convert from the locale date string?

Reply:
Removed local. I don't think any conversion needs to be done, as the date is returned from JS in the correct format of dd/mm/yyyy. Which is the format the UI is displaying the data and when used against the check_field_value (which is the main method we use to verify data) it asserts correctly.